docs/cel2/l2-operator-guide.md

```diff
@@ -24,61 +24,30 @@ Therefore, operators looking to run full archive nodes or serve requests for his
-- op-node: the consensus layer
-- op-geth: the execution layer
-
-For op-geth, there are three options to get started, ranked by ease and the level of trust required:
+Node operators who wish to run an L2 node have three options, ranked by ease and the level of trust required:
 
 1. Start an L2 node with snap sync. This option does not require running the migration script.
 2. Start an L2 node with the provided L1 chaindata.
 3. Migrate the L1 chaindata manually.
 
-Using snap sync offers a simpler and faster experience, but it cannot be used if you want to run an archive node. In that case, you will need to use an archive node snapshot or migrate your own archive data from an L1 node.
+Snap sync provides a simpler and faster setup experience. However, it is not suitable if you plan to run an archive node. In that case, you'll need to either use an archive node snapshot or migrate your own archive data from an L1 node. While both options follow a similar process, there are some differences, particularly in how you prepare the chaindata from the L1. Nonetheless, the majority of the service configuration remains the same across all options.
 
-:::info
-If you plan to migrate your own chaindata, we recommend getting familiar with the instructions and running the pre-migration 1-3 days ahead of the full migration.
-:::
+The node setup will require running three services: op-node, op-geth, and eigenda-proxy. op-node is the consensus client for the L2 node, op-geth is the execution client, and eigenda-proxy is the interface between your L2 node and [EigenDA](https://www.blog.eigenlayer.xyz/intro-to-eigenda-hyperscale-data-availability-for-rollups/). If you want to run multiple L2 nodes, you will need to run multiple instances of op-node and op-geth, but can share a single instance of eigenda-proxy.
 
-The next sections will provide more information on running each component, including the different options for op-geth.
+- [Running EigenDA Proxy](#running-eigenda-proxy)
+- [Running op-node](#running-op-node)
+- [Running op-geth](#running-op-geth)
 
-### Running EigenDA Proxy
+#### Running EigenDA Proxy
 
-:::info
 These are brief instructions for running an eigenda-proxy instance. For more detailed instructions, please refer to the [repository README](https://github.com/Layr-Labs/eigenda-proxy/tree/main?tab=readme-ov-file#deployment-guide).
-:::
-
-If you are using Kubernetes for this deployment, you can utilize our [eigenda-proxy Helm chart](https://github.com/celo-org/charts/tree/main/charts/eigenda-proxy) to simplify the process. Feel free to modify these instructions to better suit your specific needs.
-
-To run the eigenda-proxy, you can use the container image: ghcr.io/layr-labs/eigenda-proxy:v1.4.1. Alternatively, you can clone the [Layr-Labs/eigenda-proxy repository](https://github.com/Layr-Labs/eigenda-proxy) and build the proxy:
-
-```bash
-git clone https://github.com/Layr-Labs/eigenda-proxy.git
-cd eigenda-proxy
-git checkout v1.2.0
-make
```

Comment:
I found it clearer to have the choice between container and building the image before the rest of the instructions to keep them more chronological. Otherwise you have the instructions to configure and run, then the instructions to build afterwards even though you need to do that before if you want that.